src/OpenTelemetry.SemanticConventions/SemanticConventions/v1_23_1_Experimental/CloudeventsAttributes.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="CloudeventsAttributes.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+// <auto-generated> This file has been auto generated from buildscripts/semantic-conventions/templates/Attributes.cs.j2</auto-generated>
+
+using System;
+
+namespace OpenTelemetry.SemanticConventions.v1_23_1_Experimental
+{
+    /// <summary>
+    /// Describes semantic conventions for attributes in the <c>cloudevents</c> namespace.
+    /// </summary>
+    public static class CloudeventsAttributes
```

Comment:
`cloudevents`, `graphql`, `webengine`, `opentracing`, etc cannot be automatically CamelCased, so need to figure out something on the semconv side to fix it across languages

Reply:
https://github.com/open-telemetry/semantic-conventions/issues/599